src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,12 @@
+package duke.exception;
+
+/**
+ * Print error messages
+ */
+public class DukeException extends Exception {
```

Comment:
An interesting way to use DukeException class that I've seen from another PR was to have different types of DukeExceptions extending from this class with their own error messages. The programs will then throw these exceptions instead of the various java exceptions. Just sharing something I learnt! You could consider doing it too if you think that it's better 😄